front-end/src/components/helpers/ExampleTranscript.js

```diff
@@ -0,0 +1,8 @@
+export const exampleTranscript = {"data": [
+        {
+            "project_id": "[INSERT UNIQUE ID]",
+            "trace_type": "speak",
```

Comment:
Does text look exactly the same as speak? Especially the payload part

Reply:
It has some extra parts but the important one which is message is the same in both.
this is one example: 
{
        "": "408510",
        "project_id": "1",
        "turn_id": "9996a941-51d9-4a8e-840f-d4da97383055",
        "ts": "2022-03-28 15:19:55.011 UTC",
        "session_id": "4a0303a9c0b04a36f7ce9ee539d2c5c1855ebf7f60f22da1d9e2eff4338a7ae4",
        "platform": "prototype",
        "interaction_metadata": "{\"end\":false,\"locale\":\"en-US\"}",
        "trace_format": "trace",
        "trace_type": "text",
        "trace_payload": "{\"type\":\"text\",\"payload\":{\"slate\":{\"id\":\"mb17s3c5c\",\"content\":\"[{\\\"children\\\":[{\\\"text\\\":\\\"Would you like to buy this product?\\\"}]}]\"},\"message\":\"Would you like to buy this product?\"}}"
    },

Reply:
Actually I was thinking even for speak you don't even need to have the voice name. The code doesn't care. As long as it has a "message" it will add that message as a question.